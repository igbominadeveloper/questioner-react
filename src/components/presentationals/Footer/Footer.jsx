import React from 'react';
import { Link } from 'react-router-dom';
function Footer() {
  return (
    <footer data-test="footer">
      <div className="footer-content flex text-white flex-row">
        <div className="logo">
          <div className="footer-groups flex flex-column">
            <div className="footer-group-title">Questioner Logo</div>
          </div>
        </div>
        <div className="flex flex-row space-between links">
          <div className="footer-groups flex flex-column">
            <div className="footer-group-title">Questioner</div>
            <Link to="/#">
              <p>About us</p>
            </Link>
            <Link to="/#">
              <p>How it works</p>
            </Link>
            <Link to="/#">
              <p>Contact us</p>
            </Link>
          </div>
          <div className="footer-groups flex flex-column">
            <div className="footer-group-title">Account</div>
            <Link to="/register.html">
              <p>Sign up</p>
            </Link>
            <Link to="/login.html">
              <p>Sign in</p>
            </Link>
            <Link to="/password-reset.html">
              <p>Password reset</p>
            </Link>
          </div>
          <div className="footer-groups flex flex-column">
            <div className="footer-group-title">Quick Links</div>
            <Link to="/meetups.html">
              <p>All meetups</p>
            </Link>
            <Link to="/#">
              <p>Upcoming</p>
            </Link>
            <Link to="/#">
              <p>Latest</p>
            </Link>
          </div>
        </div>
      </div>
      <div className="copyright bg-white p-10">
        <p className="text-center">
          Made with <i className="fa fa-heart text-danger" />
          by &nbsp;
          <Link to="/https://github.com/igbominadeveloper" className="github">
            igbominadeveloper
          </Link>
        </p>
      </div>
    </footer>
  );
}

export default Footer;
