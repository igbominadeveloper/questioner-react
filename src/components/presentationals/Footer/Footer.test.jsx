import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

describe('Footer', () => {
  describe('Render', () => {
    it('renders without error', () => {
      const component = shallow(<Footer />);
      expect(component).toMatchSnapshot();
    });
  });
});
