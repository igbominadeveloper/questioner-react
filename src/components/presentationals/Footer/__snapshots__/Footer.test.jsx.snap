// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Render renders without error 1`] = `
<footer
  data-test="footer"
>
  <div
    className="copyright bg-white p-10"
  >
    <p
      className="text-center"
    >
      Made with 
      <i
        className="fa fa-heart text-danger"
      />
      by Â 
      <a
        className="github"
        href="https://github.com/igbominadeveloper"
        target="_blank"
      >
        igbominadeveloper
      </a>
    </p>
  </div>
</footer>
`;
